plugins/woocommerce/includes/abstracts/abstract-wc-order.php

```diff
@@ -2613,4 +2654,17 @@ public function set_cogs_total_value( float $value ) {
 			$this->set_prop( 'cogs_total_value', $value );
 		}
 	}
+
+	/**
+	 * Check if the order is a POS (Point of Sale) order.
+	 *
+	 * This method determines if an order was created via the POS REST API
+	 * by checking the 'created_via' property of the order.
+	 *
+	 * @since 9.9.0
+	 * @return bool True if the order is a POS order, false otherwise.
+	 */
+	public function is_pos_order(): bool {
```

Comment:
The class is huge at this point :sweat_smile:, let's not add this one-liner.

Reply:
As in https://github.com/woocommerce/woocommerce/pull/57573#discussion_r2071162862, I moved this `is_pos_order` to `/src/Internal/Orders/POSOrderUtil`. This could be helpful for other POS UI changes in core, like for the [order details](https://github.com/woocommerce/woocommerce/issues/56675) and [order list UI](https://github.com/woocommerce/woocommerce/issues/56674).